grammars/go.cson

```diff
@@ -509,15 +468,51 @@
                 'include': '$self'
               }
             ]
-          '3':
-            'patterns': [
-              {
-                'include': '$self'
-              }
-            ]
       }
       {
-        'match': '([a-zA-Z_]\\w*(?:,\\s*[a-zA-Z_]\\w*)*)(\\s+[\\[\\]\\*]{0,3}[a-zA-Z_]\\w*\\s*[^=].*)'
+        # Multiline variable declarations/assignments
+        'begin': '(?<=var)\\s+(\\()'
+        'beginCaptures':
+          '1':
+            'name': 'punctuation.other.bracket.round.go'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.other.bracket.round.go'
+        'patterns': [
+          {
+            # Copy of below, but without the leading var
+            # var (a *b)
+            'match': '([a-zA-Z_]\\w*(?:,\\s*[a-zA-Z_]\\w*)*)(\\s+[\\[\\]\\*]{0,3}[a-zA-Z_]\\w*\\s*[^=].*)'
+            'captures':
+              '1':
+                'patterns': [
+                  {
+                    'match': '[a-zA-Z_]\\w*'
+                    'name': 'variable.other.declaration.go'
+                  }
+                  {
+                    'include': '#delimiters'
+                  }
+                ]
+              '2':
+                'patterns': [
+                  {
+                    'include': '$self'
+                  }
+                ]
+          }
+          {
+            'include': '$self'
+          }
+        ]
+      }
+      {
+        # var a
+        # var a string
+        # var a, b string
+        # var a []string
+        'match': '(?<=var)\\s+([a-zA-Z_]\\w*(?:,\\s*[a-zA-Z_]\\w*)*)(\\s+[\\[\\]\\*]{0,3}[a-zA-Z_]\\w*\\s*[^=].*)'
```

Comment:
I just didn't know what the `<=` means, so a `[^\\w]` is added here.
It should be better if there are modifications to `(?<=var)`  so that it can match `var` at word boundary.